src/igvfd/audit/file.py

```diff
@@ -70,3 +70,26 @@ def audit_file_format_specifications(value, system):
                 f'with `document_type` {doc_type}.'
             )
             yield AuditFailure(audit_message.get('audit_category', ''), f'{detail} {audit_message.get("audit_description", "")}', level=audit_message.get('audit_level', ''))
+
+
+@audit_checker('ModelFile', frame='object')
+@audit_checker('SequenceFile', frame='object')
+def audit_external_identifiers(value, system):
+    '''
+    [
+        {
+            "audit_description": "Externally hosted files are expected to have identifiers from external resources.",
+            "audit_category": "missing identifiers from external resources",
```

Comment:
I think this is OK, but wondering if it would be reasonable to shorten to "missing dbxrefs"? Or do you think that is ambiguous (because it doesn't mention that it's due to being external)?

Reply:
yeah I didn't think too much on the this, I think 'missing dbxrefs' would work and I will also update the description.